src/components/TestingModelTable.tsx

```diff
@@ -44,12 +43,9 @@ const headings = [
   },
 ];
 
-const TestingModelTable = ({
-  prediction,
-}: {
-  prediction: PredictionResult | undefined;
-}) => {
-  const { detected, confidences } = prediction ?? {};
+const TestingModelTable = () => {
+  // const predictionResult = useStore((s) => s.predictionResult);
```

Comment:
Remove this?

Reply:
Did it, but didn't push it... [419ae4d](https://github.com/microbit-foundation/ml-trainer/pull/489/commits/419ae4ddaa183f799c6f15f16734b11aeeec5c67)